priv/templates/boot.eex

```diff
@@ -208,6 +208,15 @@ _replace_os_vars() {
         }1' < "$1" > "$2"
 }
 
+# Private. Run hooks from directory.
+_run_hooks_from_dir() {
+   if [ -d "$1" ]; then
+          for s in $(find "$1" -name "*.sh" -print | sort); do
```

Comment:
Works for me.

I did some additional digging; I am used to (defensively) skip any files not starting with alphanum. `vi` and `rsync` used to create `.yourfile.sh` or `.#yourfile.sh`, modern versions of both also add a suffix, so this is less of a concern.